saleor/webhook/transport/synchronous/circuit_breaker/storage.py

```diff
@@ -118,3 +131,11 @@ def register_event_returning_count(
         except (RedisError, IndexError):
             logger.warning(self.WARNING_MESSAGE, exc_info=True)
             return 0
+
+    def close_breaker(self, app_id: int):
```

Comment:
I understand the use of `*` (wildcard) to obtain keys because it protects us in future, had we add any more counters in the storage (apart from the current main key, `total` and `error`).

Although I'd recommend having a fixed list of keys to delete here (main key, `total` key and `error` key). We would not make two requests to Redis and it'd be much more clearer to the person who reads the code what is actually happening.

Related but nothing actionable for this part of the code. If we do more than more one operation with Redis consider using pipelines.